src/main.cpp

```diff
@@ -74,6 +81,54 @@ int main(int argc, char *argv[]){
             return 2;
         }
     }
+
```

Comment:
Why are you generating password here in main.cpp? The purpose of the class we are making is to abstract away this process in a class so that we don't have to do this in main.cpp. Your PR defeats the whole purpose of the exercise.